MapboxNavigation/NavigationViewController.swift

```diff
@@ -546,6 +546,10 @@ extension NavigationViewController: NavigationServiceDelegate {
             mapViewController?.mapView.updateCourseTracking(location: location, animated: true)
         }
         
+        if waypointStyle != .annotation, progress.currentLegProgress.distanceRemaining < 500 {
```

Comment:
I think Android is currently using threshold of [250 meters](https://github.com/mapbox/mapbox-navigation-android/pull/3515/files#diff-e8bba261553ad36e3c4c76f514f7f798R136).